spec/menu_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'menu'
+
+describe Menu do
+
+  subject(:menu) { Menu.new }
```

Comment:
When describing a class `subject` is automatically a `.new` instance of the class